cpp/cmake_modules/FindUcx.cmake

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+find_package(PkgConfig QUIET)
+pkg_check_modules(Ucx IMPORTED_TARGET ucx)
```

Comment:
IIRC later versions of UCX also ship CMake files

Reply:
Thanks for remind. Will check how to use it.

Reply:
We can just use `find_package(ucx ...)`. And we don't need to add this file.

Reply:
Done